### ISSUE TITLE
max-len rule reports undefined as error column

### ISSUE THREAD

**Issue by lo1tuma** [eslint/eslint#2926]
The `max-len` rule doesn’t report the correct column. You can reproduce this in the eslint online demo with the following example:

```js
/*eslint max-len: [2, 10, 4] */
var foo = "bar";
```

As you can see in the error message `2:undefined - Line 2 exceeds the maximum line length of 10.` the column is `undefined`.
On current master the column number is `NaN` because of the recent change to 1-based column numbers.

**Comment by lo1tuma** [comment]
working on this
